wgpu/src/lib.rs

```diff
@@ -858,11 +858,17 @@ pub enum ShaderSource<'a> {
         defines: naga::FastHashMap<String, String>,
     },
     /// WGSL module as a string slice.
+    #[cfg(feature = "wgsl")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "wgsl")))]
     Wgsl(Cow<'a, str>),
     /// Naga module.
     #[cfg(feature = "naga")]
     #[cfg_attr(docsrs, doc(cfg(feature = "naga")))]
     Naga(Cow<'static, naga::Module>),
+    /// Dummy variant because `Naga` doesn't have a lifetime and without enough active features it
+    /// could be the last one active.
+    #[doc(hidden)]
```

Comment:
Any reason this can't be conditional?

Reply:
It could, but it didn't seem to me to give any advantage, I would have to put this conditional on every `match` statement too now. Should I still add the conditional?

Reply:
Oh, true - eh whatever